src/mpl/src/gpu/mpl_gpu_hip.c

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
Hip is not supported as for now. Can you remove this?

Reply:
removed